deploy/template.yaml

```diff
@@ -1299,6 +1320,155 @@ Resources:
               Bool:
                 "aws:SecureTransport": true
 
+  #
+  # Metrics and Alarms
+  #
+
+  # API Gateway Alarms
+
+  # 4xx Status Code
+  ApiGateway4xxStatusAlarm:
+    Condition: IsNotDevelopment
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmName: !Sub "${AWS::StackName}-ApiGateway4xxStatusAlarm"
+      AlarmDescription: "Trigger the alarm if errorThreshold exceeds 1% with the minimum of 10 invocations in 2 out of the last 5 minutes"
+      ActionsEnabled: false  # disabled until we're ready to invoke alarm actions
+      AlarmActions:
+        - !Sub "{{resolve:ssm:/${AlertingStackName}/SNS/HighAlertNotificationTopic/ARN}}"
+      OKActions:
+        - !Sub "{{resolve:ssm:/${AlertingStackName}/SNS/HighAlertNotificationTopic/ARN}}"
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      EvaluationPeriods: 5
+      DatapointsToAlarm: 2
+      Threshold: 1
+      TreatMissingData: missing
```

Comment:
Is this what happens in the absence of data?
What does missing do?

Reply:
As per documentation [here](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/AlarmThatSendsEmail.html#alarms-and-missing-data):

If all data points in the alarm evaluation range are missing, the alarm transitions to INSUFFICIENT_DATA.

Reply:
So yes that's right, the alarm goes into an INSUFFICIENT_DATA state in the absence of data for a given evaluation period.